src/extension/features/accounts/split-transaction-auto-fill-payee/index.js

```diff
@@ -0,0 +1,25 @@
+import { Feature } from 'toolkit/extension/features/feature';
+import { isCurrentRouteAccountsPage } from 'toolkit/extension/utils/ynab';
+
+export class SplitTransactionAutoFillPayee extends Feature {
+  shouldInvoke() {
+    return isCurrentRouteAccountsPage() && $('.ynab-grid-split-add-sub-transaction').length !== 0;
+  }
+
+  invoke() {
+    const cells = $('.is-editing .ynab-grid-cell-payeeName .ember-text-field').toArray();
+    cells.forEach((cell, i) => {
+      if (i !== 0 && !$(cell).data('tk-auto-filled-payee')) {
+        $(cell).data('tk-auto-filled-payee', true);
```

Comment:
Could use `cell.dataset`, but since jQuery is being used everywhere else, figured I'd use that as well to be consistent.